extract/dcinside/refactor3.py

```
@@ -0,0 +1,541 @@
+        
+        # 수집된 링크를 방문하며 html 소스 저장
+        fail_count = 0
+        for i, post in enumerate(self.post_link):
+
+            parsed_source = self.get_html_of_post(driver, post['url'])
+            res_json = self.html_parser(driver, post, parsed_source)
+            
+            logger.info(f"💿 ⏎ Saving...[{i+1} / {len(self.post_link)}]")
+            try:
+                self.save_json(res_json, post)
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)